jvm/src/main/scala/io/glutenproject/expression/InSetOperatorTransformer.scala

```diff
@@ -54,32 +54,35 @@ object InSetOperatorTransformer {
   def toTransformer(value: Expression,
                     leftNode: ExpressionNode,
                     values: Set[Any]): ExpressionNode = {
-    val expressionNodes = new util.ArrayList[ExpressionNode]()
-    val listNode = value.dataType match {
+    val expressionNodes = value.dataType match {
```

Comment:
expressionNodes -> expressionNode

Reply:
Could you explain this change a bit? Because it is an arrayList of ExpressionNode, so I guess expressionNodes may be better.

Reply:
ok make sense